content/specific-skill-success-criteria/file-and-directory-naming/_index.md

```diff
@@ -0,0 +1,61 @@
+---
+_db_id: 
+content_type: topic
+ready: true
+title: Naming modules and packages 
+---
+
+## Naming modules and packages 
+
+Files are refered to as modules and directories as folders or packages.
+
+Naming conventions are rules which enable the titling of electronic and physical folders, documents and records in a consistent and logical way.
+This ensures that the correct records can be located, identified and retrieved from a filing system in a timely fashion, and that they are stored 
+in an appropriate secure location. 
+
+## Benefits of naming conventions
+
+Naming records consistently, logically and in a predictable way will distinguish similar records from one another at a glance, and by doing so will 
+facilitate the storage and retrieval of data. Through consistency and the application of logical standards, we benefit from secure storage, and the 
+ability to locate and access information.
```

Comment:
This section seems like a repetition of the same stuff we have above. Don't think we need it; **Naming modules and packages** already mentions consistency and logic, and why it's needed.